tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/InstrumentationTests.cs

```diff
@@ -132,6 +132,40 @@ public async Task InstrumentsDotNetTest()
         }
 #endif
 
+        [SkippableFact]
+        [Trait("RunOnWindows", "True")]
+        public async Task WhenUsingRelativeTracerHome_InstrumentsApp()
+        {
+            SetEnvironmentVariable("DD_INTERNAL_WAIT_FOR_DEBUGGER_ATTACH", "1");
```

Comment:
Is this by design? If so, probably needs a comment as it looks quite odd.

Reply:
ah yes it's something either I or Andrew forgot there, it needs to be removed indeed 😅